### ISSUE TITLE
Warp Box

### ISSUE THREAD

**Issue by kixelated** [kixelated/warp-draft#7]
Currently there's a `warp` box containing metadata about the upcoming init or segment data. However it's not clear what should be in this box, or if it needs to exist at all.

You can identify if it's an init or media segment based on the first atom. `ftyp` versus `styp`. Should we have a single message instead of `init` versus `segment`?

You can identify the timestamp of the first sample by parsing the first `moof`. Should we expose the timestamp of the first sample at the warp layer? I think so so we can offset timestamps (stitching) without modifying the entire container.

You can identify the track by using the TrackID littered throughout the MP4. Should we expose the track in the warp layer so middleware can parse/route it easier?

The only thing that you can't identify is which init segment to use. This could be avoided by using multiple tracks rather than reinitializing an existing track.

Basically, how much do we expose at the Warp (network) layer versus how much should be parsed at the media (MP4) layer?

**Comment by kixelated** [comment]
Removed.
